stepper-2.0.md

```diff
@@ -59,42 +59,58 @@ Protocol
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
-2  stop command                            (0x01)
+2  zero command                            (0x01)
 3  device number                           (0-9)
 4  END_SYSEX                               (0xF7)
 ```
 
-**Stepper step**
+**Stepper step (relative move)**
+
+Position is specified as a 32-bit unsigned long. Speed is specified as a
+16-bit unsigned int.
+
 ```
 0  START_SYSEX                             (0xF0)
 1  Stepper Command                         (0x62)
 2  step command                            (0x02)
 3  device number                           (0-9)
 4  direction                               (0-1) (0x00 = CW, 0x01 = CCW)
-5  num steps byte1 LSB
-6  num steps byte2
-7  num steps byte3 MSB                     (21 bits (2,097,151 steps max))
-8  num speed LSB                           (steps per second)
-9  num speed MSB
-10 END_SYSEX                               (0xF7)
+5  num steps, bits 0-6
```

Comment:
I guess it gets interesting when considering the properties of numbers various in the languages that a Firmata client library has been created for. With JavaScript for example, Numbers are 64-bits so if you wanted to send a 16 bit and shifted 16 bits into 3 Firmata bytes, you would not get the signed bit since that would be the the MSB of the 64-bit Number (different of course if using typed Arrays in JS). So I guess a separate sign byte (same as direction byte) may be necessary, or the MSB bit would need to be moved to the MSB of the size expected by Firmata.